cmd/cmd.go

```diff
@@ -53,7 +53,7 @@ func NewCmdRoot(streams genericclioptions.IOStreams) *cobra.Command {
 				os.Exit(1)
 			}
 
-			if !skipVersionCheck {
+			if !skipVersionCheck || cmd.Use == "upgrade" {
```

Comment:
Maybe also `|| cmd.Use == "version"` as well?  I think version already tells you if you're not on the latest as part of the output.

Reply:
Is it possible (and does it make sense) to make this a constant rather than an anonymous string?

Reply:
SGTM

Reply:
Also just now realizing this should be "&&" but we'll ignore that :)